### PR TITLE
Creating a database access library

### PR DESCRIPTION
**Pull request by fatcat2**

### CODE REVIEW

File: biit_server/database.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        except Exception:
+            return False
+
+    def query(self, attr, operation, value):
+        """Helper function to query documents based on parameters"""
+        try:
+            results = self.collection_ref.where(attr, operation, value).stream()
+            return [value for value in results]
+        except Exception:
+            return False
```
Comment (stephend017) [inline biit_server/database.py:33]: Can you include some docstrings or atleast type hints so we dont need to bother you about how to use this @fatcat2

File: tox.ini [hunk 2]
```diff
@@ -7,6 +7,8 @@ envlist = py38
 deps =
 	pytest
 	black
+	mock-firestore
+	-rrequirements.txt
```
Comment (stephend017) [inline tox.ini:11]: is this correct? `-rrequirements.txt` without the spaces?

**Review by stephend017 — APPROVED** [review]
Would like to see some documentation on how to use but other than that it looks good to me

File: tox.ini (hunk 2 above)
Comment (fatcat2) [reply tox.ini:11]: Yeah I googled it lol

Comment (fatcat2) [reply tox.ini:11]: It was in the official docs

Comment (stephend017) [reply tox.ini:11]: interesting. thanks for checking

**Comment by sotails** [comment]
Looks good to me, just want some documentation in case others may need to make changes.

**Review by sotails — APPROVED** [review]

**Pull request merged**